Core/src/main/java/com/plotsquared/core/events/PlayerTeleportToPlotEvent.java

```diff
@@ -57,14 +60,36 @@ public TeleportCause getCause() {
     }
 
     /**
-     * Get the from location
+     * Get the location, from where the teleport was triggered
+     * (the players current location when executing the home command for example)
      *
      * @return Location
      */
     public Location getFrom() {
         return this.from;
     }
 
+    /**
+     * Gets the currently applied {@link LocationTransformer} or null, if none was set
+     *
+     * @return LocationTransformer
+     * @since TODO
+     */
+    public @Nullable LocationTransformer getLocationTransformer() {
```

Comment:
Is it not easier just to use `Function<Location, Location>`?

Reply:
or even `UnaryOperator<Location>`